Documentation/Signoffs/DAQ-Subsystem-Signoff.md

```diff
@@ -0,0 +1,120 @@
+# DAQ Signoff
+
+## Functionality
+
+The primary function of the data acquisition (DAQ) is capturing and displaying relevant testing data in real-time. The DAQ resolves this constraint through a process containing signal conditioners, signal converters, a controller, and software. Through the execution of correctly applying these systems, accurate, real-time data will be derived to support the generation of a model.
+
+## Constraints
+|No.        |Constraints   |Origin   |
+|---|---------------------|--------|
+|3|Capture current and voltage supplied to the drive motor along with vehicle speed, torque, horsepower, and RPM and display all results.         |Design Constraint |
+|5|Comply with all relevant codes and standards listed in II. Ethical, Professional, and Standard Considerations.       |Ethics Constraint|
+|16|Analog input pins must be able to read a 0-5 V or 4-20 mA output from the sensors.       |Derived from further considerations|
```

Comment:
with what resolution? The resolutions listed below seem to be for the motor not general analog sensors.